src/Catalyst.Core.Modules.Consensus.Tests/UnitTests/Deltas/DeltaBuilderTests.cs

```diff
@@ -65,14 +71,17 @@ public sealed class DeltaBuilderTests
         private readonly CoinbaseEntry _zeroCoinbaseEntry;
         private readonly IDeltaCache _cache;
         private readonly IDateTimeProvider _dateTimeProvider;
+        private readonly IStateProvider _stateProvider;
+        private readonly IDeltaExecutor _deltaExecutor;
+        private readonly ICryptoContext _cryptoContext;
         private readonly ILogger _logger;
         private readonly IPeerSettings _peerSettings;
 
         public DeltaBuilderTests()
         {
             _hashProvider = new HashProvider(HashingAlgorithm.GetAlgorithmMetadata("blake2b-256"));
 
-            _random = new Random();
+            _random = new Random(1);
```

Comment:
notice we needed a predictable seed for testing

Reply:
👍